Changed Regex of 3rd textString (#38944)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/basic-css/import-a-google-font.english.md

## Instructions
<section id='instructions'>

> Import the Lobster font to your web page. Then use an <code>h2</code> element selector to create a <code>font-family</code> CSS rule that uses the Lobster font to ensure it will be applied to your <code>h2</code> element.

Reviewer: ```suggestion
Import the Lobster font to your web page. Then, use an element selector to set <code>Lobster</code> as the <code>font-family</code> for your <code>h2</code> element.
```

Reviewer 2: I'm fine with this if you are @RandellDawson

Reviewer 3: @moT01 Go ahead and commit that suggestion or create a new commit.  @Sky020 and I will be the reviewers/approvers for this PR since you are adding commits.
